newton/_src/geometry/sdf_utils.py

```
@@ -0,0 +1,256 @@
+        is_occupied = signed_distance > threshold[0]
+    tile_occupied[tid] = is_occupied
+
+
+def compute_sdf(
+    shape_thickness: float,
+    mesh_src,
+    narrow_band_distance: Sequence[float] = (-0.1, 0.1),
+    margin: float = 0.05,
+    max_dims: int = 64,
+    verbose: bool = False,
+    device=None,
```

Ruff (0.14.6)

Unused function argument: `device`

(ARG001)